bot/utils/get_epoch.py

```diff
@@ -0,0 +1,8 @@
+from datetime import datetime, timedelta, timezone
+
+
+def get_x_days_ago_UTC_timestamp(x: int):
```

Comment:
Please add the return type `def get_x_days_ago_UTC_timestamp(x: int) -> int` and the previous name suggestion `get_days_ago_UTC_timestamp` is still available 🙂